src/main/groovy/org/shipkit/internal/gradle/NotesGeneration.java

```diff
@@ -9,6 +9,7 @@
 //TODO expose as public API
 public class NotesGeneration {
```

Comment:
NotesGeneration along with NotableReleaseNotesFetcherTask is dead code. Can you get rid of it? This will help keeping the codebase clean! Please also remove other stuff connected to those 2 classes (if you find any :)